core/include/detray/geometry/shapes/line.hpp

```diff
@@ -181,6 +181,6 @@ class line {
 // Radial crossection, boundary check in polar coordiantes
 using straw_tube = line<false>;
 // Square crossection, boundary check in cartesian coordiantes
-using wire_cell = line<true>;
+using drift_chamber_cell = line<true>;
```

Comment:
BTW why don't you just call them `line_circular` and `line_rectangular`?  `straw_tube` or `drift_chamber_cell` sounds a bit too specific to be defined in shape directory. This naming also does not consider vertex reconstruction

I guess that we can also define `drift_chamber_cell` or `straw_tube` in detector metadata

Reply:
Done